src/picongpu/CMakeLists.txt

```diff
@@ -359,10 +359,29 @@ if(ADIOS_FOUND)
 
     include_directories(SYSTEM ${ADIOS_INCLUDE_DIRS})
     set(LIBS ${LIBS} ${ADIOS_LIBRARIES})
+    add_definitions(-DADIOS_VERSION="${ADIOS_VERSION}")
```

Comment:
Only necessary for ADIOS < 1.9.0, see https://github.com/ornladios/ADIOS/commit/61cce400018a125933b85ae1b22555ccb2954d80